e2e/e2e_test.go

```diff
@@ -48,12 +57,22 @@ var _ = Describe("Whereabouts functionality", func() {
 	Context("Test setup", func() {
 		var (
 			clientInfo   *ClientInfo
+			envVars      *envVars
```

Comment:
the variable name is shadowing the variable type - please rename the variable type to something else - `testConfig` for instance.